File: 1012.cpp
// André Nascimento
// GitHub: github.com/AndreNasci
// Linkedin: linkedin.com/in/andre-nascimento-a01095185

#include <bits/stdc++.h>

using namespace std;

int main()
{
    double A, B, C;
    double pi = 3.14159;

    cin >> A >> B >> C;

    cout << fixed << setprecision(3);
    cout << "TRIANGULO: " << A * C / 2 << endl;
    cout << "CIRCULO: " << pi * C * C << endl;
    cout << "TRAPEZIO: " << (A + B) * C / 2 << endl;
    cout << "QUADRADO: " << B * B << endl;
    cout << "RETANGULO: " << A * B << endl;

    return 0;
}

File: Python/1012.py
"""
André Nascimento
GitHub: github.com/AndreNasci
Linkedin: linkedin.com/in/andre-nasci/
"""

A, B, C = input().split()
A = float(A)
B = float(B)
C = float(C)

pi = 3.14159 

print(f"TRIANGULO: {(A * C / 2):.3f}")
print(f"CIRCULO: {(pi * C ** 2):.3f}")
print(f"TRAPEZIO: {((A + B) * C / 2):.3f}")
print(f"QUADRADO: {(B ** 2):.3f}")
print(f"RETANGULO: {(A * B):.3f}")
